api/include/opentelemetry/trace/span_id.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+namespace opentelemetry {
+namespace trace {
+
+class SpanId {
+ public:
+  // The size in bytes of the SpanId.
+  static constexpr size_t kSize = 8;
```

Comment:
Curious:
1. Do we need this to be public?
2. Which naming style are we following?

Reply:
IMO it doesn't _have to_ be public, but my reasoning is that span_id is a rigidly defined part of the otel data model and having a size constant is useful.

There would be e.g. `SpanId::kSize` and also `TraceId::kSize`